### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by interrogator**

### CODE REVIEW

File: loggo/loggo.py [hunk 1]
```diff
@@ -77,58 +47,37 @@ class Loggo(object):
     - do_print: print logs to console
     - do_write: write logs to file
     - line_length: max length for console printed string
-    - private_data: key names that should be filtered out of logging. if not set,
+    - private_data: key names that should be filtered out of logging
```
Comment (hukkinj1) [inline loggo/loggo.py]: this removal is probably a mistake

**Comment by hukkinj1** [comment]
A changelog would be nice to have for all backwards incompatible changes. At least the removal of LoggedException is such a change. Maybe copy the base for it from py-utils or cilib?

File: loggo/loggo.py [hunk 2]
```diff
@@ -77,58 +47,37 @@ class Loggo(object):
...
-        self.do_colour = config.get('colour', False)
         self.ip = config.get('ip', None)
         self.port = config.get('port', None)
         self.do_print = config.get('do_print', False)
         self.do_write = config.get('do_write', False)
         self.logfile = config.get('logfile', './logs/logs.txt')
         self.line_length = config.get('line_length', 200)
         self.obscured = config.get('obscure', '[PRIVATE_DATA]')
-        self.private_data = config.get('private_data', DEFAULT_PRIVATE_KEYS)
+        self.private_data = config.get('private_data')
```
Comment (hukkinj1) [inline loggo/loggo.py]: Would be great to have consistency here with `self.ip = config.get('ip', None)` et al above. I think it's pretty clear to everyone that `None` is default so I'd prefer omitting it like on this line

File: loggo/loggo.py [hunk 3]
```diff
@@ -77,58 +47,38 @@ class Loggo(object):
     - do_print: print logs to console
     - do_write: write logs to file
     - line_length: max length for console printed string
-    - private_data: key names that should be filtered out of logging. if not set,
+    - private_data: key names that should be filtered out of logging. when not
+    provided, nothing is censored
       some sensible defaults are used
```
Comment (hukkinj1) [inline loggo/loggo.py]: this line probably useless now?

Comment (interrogator) [reply loggo/loggo.py]: nope, it's still in use

**Pull request merged**